src/website/src/app/documentation/demos/datepicker/demos/datepicker-api.demo.html

```diff
@@ -1,10 +1,12 @@
 <h4>Basic API</h4>
 <p>
     To use the date picker, add the <code class="clr-code">clrDate</code> directive on an
-    <code class="clr-code">input</code> field.
+    <code class="clr-code">input</code> field and add it between <code class="clr-code">clr-date-container</code> tags.
```

Comment:
Would it sound more natural to describe how to use the clrDate directive and then describe its relationship to the parent container? e.g - 

```To use the date picker, add the <code class="clr-code">clrDate</code> directive to an <code class="clr-code">input</code> field.	Then, place the input inside the <code class="clr-code">clr-date-container</code> container element.```

Reply:
Thank you for updating. 
lgtm - please squash your commits and we can get this merged.